enclave_wrangler/datasets.py

```diff
@@ -191,62 +131,23 @@ def transform_dataset__concept_relationship(dataset_name: str) -> pd.DataFrame:
     """Transformations to concept_relationship.csv"""
     df = pd.read_csv(os.path.join(CSV_DOWNLOAD_DIR, dataset_name + '.csv'), keep_default_na=False).fillna('')
     df = transforms_common(df, dataset_name)
-
-    # JOIN & Filter
-    try:
-        csm_df = pd.read_csv(
-            os.path.join(CSV_TRANSFORM_DIR, 'concept_set_members.csv'), keep_default_na=False).fillna('')
-        # JOIN
-        cr_csm = df.merge(csm_df, left_on='concept_id_1', right_on='concept_id')
-        # Filter
-        df = df[df['concept_id_1'].isin(cr_csm['concept_id_1'])]
-    except FileNotFoundError:
-        print('Warning: Tried transforming concept_relationship.csv, but concept_set_members.csv must be downloaded '
-              'and transformed first. Try running again after this process completes. Eventually need to do these '
-              'transformations dependency ordered fashion.', file=sys.stderr)
-
     # Filter
     df2 = df[df['relationship_id'] == 'Subsumes']
     df2.to_csv(os.path.join(CSV_TRANSFORM_DIR, 'concept_relationship_subsumes_only.csv'), index=False)
-
     return df
 
 
 def transform_dataset__concept(dataset_name: str) -> pd.DataFrame:
     """Transformations to concept.csv"""
     df = pd.read_csv(os.path.join(CSV_DOWNLOAD_DIR, dataset_name + '.csv'), keep_default_na=False).fillna('')
     df = transforms_common(df, dataset_name)
-
-    # JOIN
-    try:
-        csm_df = pd.read_csv(
-            os.path.join(CSV_TRANSFORM_DIR, 'concept_set_members.csv'), keep_default_na=False).fillna('')
-        concept_csm = df.merge(csm_df, on='concept_id')
-        df = df[df['concept_id'].isin(concept_csm['concept_id'])]
-    except FileNotFoundError:
-        print('Warning: Tried transforming concept.csv, but concept_set_members.csv must be downloaded and transformed '
-              'first. Try running again after this process completes. Eventually need to do these transformations '
-              'dependency ordered fashion.', file=sys.stderr)
-
     return df
 
 
 def transform_dataset__concept_ancestor(dataset_name: str) -> pd.DataFrame:
     """Transformations to concept_ancestor.csv"""
     df = pd.read_csv(os.path.join(CSV_DOWNLOAD_DIR, dataset_name + '.csv'), keep_default_na=False).fillna('')
     df = transforms_common(df, dataset_name)
-
-    # JOIN
-    try:
-        csm_df = pd.read_csv(
-            os.path.join(CSV_TRANSFORM_DIR, 'concept_set_members.csv'), keep_default_na=False).fillna('')
-        ca_csm = df.merge(csm_df, left_on='ancestor_concept_id', right_on='concept_id')
```

Comment:
@Sigfried This is where my vocab refresh was failing. There was this transform function that needed `concept_set_members.csv` as a dependency, but it was not pulled down through Git LFS, nor downloaded.

However I am pretty sure we don't want to do this operation anymore. Basically, several of these vocab tables were getting filtered based on the code set and concept IDs that appear in the `concept_set_members` and `code_sets` tables. I think we were doing this in order to decrease the size of these files. Perhaps this was important pre-Postgres, and this still made sense when we were refreshing via the datasets API. But now that we're refreshing via objects API, it does not make sense to do this filtering operation. So I have removed this filtering code from several functions.

I'm re-running the action now, and I think that should fix it.